internal/opcua/util.go

```diff
@@ -145,20 +145,20 @@ func pemBlockForKey(priv interface{}) (*pem.Block, error) {
 }
 
 //revive:disable-next-line
-func (o *OpcUA) generateClientOpts(endpoints []*ua.EndpointDescription) ([]opcua.Option, error) {
+func (o *OpcUAClient) generateClientOpts(endpoints []*ua.EndpointDescription) ([]opcua.Option, error) {
```

Comment:
Could you add in the comment why revive is disabled? What does it say when not disabled?

Reply:
I think this was left over from before. I removed it.